src/index.js

```diff
@@ -29,39 +33,37 @@ activeImage();
 
 leftArrow.onclick = () => {
   removeClass();
-  let index = selectedTargetID - 1;
-  displayImg.src = images[index - 1];
+  displayImg.src = setImageSrc(2);
   selectedTargetID--;
   if (selectedTargetID === 0) {
     selectedTargetID = 3;
-    displayImg.src = images[selectedTargetID - 1];
+    displayImg.src = setImageSrc(1);
   }
   activeImage();
 };
 
 rightArrow.onclick = () => {
   removeClass();
-  let index = selectedTargetID - 1;
-  displayImg.src = images[index + 1];
+  displayImg.src = setImageSrc(0);
   selectedTargetID++;
   if (selectedTargetID === 4) {
     selectedTargetID = 1;
-    displayImg.src = images[selectedTargetID - 1];
+    displayImg.src = setImageSrc(1);
   }
   activeImage();
 };
 
-for (const element of selectedImg) {
+for (const element of thumbnailElements) {
   element.onclick = (element, selectedTarget) => {
     removeClass();
     selectedTarget = element.currentTarget;
     selectedTargetID = Number(selectedTarget.id);
-    displayImg.src = images[selectedTargetID - 1];
+    displayImg.src = setImageSrc(1);
```

Comment:
how come the number in here is always passed as `1`? Shouldn't it be `selectedTargetID`? Also instead of always subtracting 1 in the `setImageSrc` function, why not just pass the actual array index number (0 through 2)?

Reply:
Lines 47, 36 have different values. Other than the for loops the rest of the functions do not have access to an index. They rely on the global ID variable.

Reply:
I added index to the for of loops to remove the function.

Reply:
I made changes to the code for the  setImageSrc function to take no params.

Reply:
Alright dope. Let's keep iterating on this. 

How can you make a function that does the incrementing/decrementing, setting the activeImg, etc and reuse it for all clicks?